Dockerfile.dev

```diff
@@ -0,0 +1,22 @@
+ARG PARENT_VERSION=latest-22
```

Comment:
this should be incorporated into the main docker file and multistage builds used, just use two targets, a default and a dev one see https://docs.docker.com/build/building/multi-stage/